Elm.gitignore

```diff
@@ -0,0 +1,2 @@
+elm-stuff/*
+!elm-stuff/exact-dependencies.json
```

Comment:
I took in #1610 which doesn't have this rule - could you tell me about what this is for any why it should be in version control?